### PR TITLE
menu: add uadelall to delete all user agents

### PR DESCRIPTION
**Pull request by philippbachmann08**
Allows to delete all user agents at once from within baresip.

### CODE REVIEW

File: modules/menu/static_menu.c [hunk 1]
```diff
@@ -719,6 +719,27 @@ static int cmd_ua_delete(struct re_printf *pf, void *arg)
 }
 
 
+static int cmd_ua_delete_all(struct re_printf *pf, void *unused)
+{
+	struct ua *ua = NULL;
+
+	(void)pf;
```
Comment (cspiel1) [inline modules/menu/static_menu.c:726]: pf is used below

**Review by cspiel1 — CHANGES_REQUESTED** [review]
remove the void cast

File: modules/menu/static_menu.c (hunk 1 above)
Comment (philippbachmann08) [reply modules/menu/static_menu.c]: thanks, removed it

**Comment by philippbachmann08** [comment]
Not sure how to interpret the log of the failed build, can the build be restarted?

**Comment by cspiel1** [comment]
> Not sure how to interpret the log of the failed build, can the build be restarted?

I think that I have to resolve the "request". I try to.

**Review by cspiel1 — APPROVED** [review]

**Comment by alfredh** [comment]
please go ahead :)

**Pull request merged**